addon/components/paper-switch.js

```diff
@@ -84,12 +92,13 @@ export default BaseFocusable.extend(RippleMixin, ProxiableMixin, ColorMixin, {
   },
```

Comment:
Instead of using jquery's `css()` and `removeAttr()` functions on `.md-thumb-container`, let's use a computed property that returns the style and bind it to the style attribute on the element like 

```hbs
<div class="md-thumb-container" style={{thumbContainerStyle}}>
  <!-- inner elements -->
</div>
```